File: TCDNew-master/src/ui.wince/Cen.Wms.Client/Common/Urls.cs
using Cen.Wms.Client.Services;

namespace Cen.Wms.Client.Common
{
    public class Urls
    {
        private const string TimeRead =                             "time/read";
        private const string UserLogin =                            "auth/login";
        private const string FacilityListSimpleReadByPerson =       "facility/list/simple/by_person";
        private const string FacilityConfigGet =                    "facility/config/get";
        private const string PacHeadReadByBarcode =                 "pac/head/read";
        private const string PurchaseTaskListByPerson =             "purchase/task/list/by_person";
        private const string PurchaseTaskCreateFromPacs =           "purchase/task/create/from_pacs";
        private const string PurchaseTaskUpdateRead =               "purchase/task/update/read";
        private const string PurchaseTaskLineReadByBarcode =        "purchase/task/line/read/by_barcode";
        private const string PurchaseTaskLineUpdatePost =           "purchase/task/line/update/post";
        private const string PurchaseTaskFinish =                   "purchase/task/finish";

        public static string TimeReadUrl { get { return GStateProvider.Instance.SettingsApp.WMSServiceBaseAddress + TimeRead; } }
        public static string UserLoginUrl { get { return GStateProvider.Instance.SettingsApp.WMSServiceBaseAddress + UserLogin; } }
        public static string FacilityListSimpleReadByPersonUrl { get { return GStateProvider.Instance.SettingsApp.WMSServiceBaseAddress + FacilityListSimpleReadByPerson; } }
        public static string FacilityConfigGetUrl { get { return GStateProvider.Instance.SettingsApp.WMSServiceBaseAddress + FacilityConfigGet; } }
        public static string PacHeadReadByBarcodeUrl { get { return GStateProvider.Instance.SettingsApp.WMSServiceBaseAddress + PacHeadReadByBarcode; } }
        public static string PurchaseTaskListByPersonUrl { get { return GStateProvider.Instance.SettingsApp.WMSServiceBaseAddress + PurchaseTaskListByPerson; } }
        public static string PurchaseTaskCreateFromPacsUrl { get { return GStateProvider.Instance.SettingsApp.WMSServiceBaseAddress + PurchaseTaskCreateFromPacs; } }
        public static string PurchaseTaskUpdateReadUrl { get { return GStateProvider.Instance.SettingsApp.WMSServiceBaseAddress + PurchaseTaskUpdateRead; } }
        public static string PurchaseTaskLineReadByBarcodeUrl { get { return GStateProvider.Instance.SettingsApp.WMSServiceBaseAddress + PurchaseTaskLineReadByBarcode; } }
        public static string PurchaseTaskLineUpdatePostUrl { get { return GStateProvider.Instance.SettingsApp.WMSServiceBaseAddress + PurchaseTaskLineUpdatePost; } }
        public static string PurchaseTaskFinishUrl { get { return GStateProvider.Instance.SettingsApp.WMSServiceBaseAddress + PurchaseTaskFinish; } }
    }
}

File: TCDNew-master/src/ui.android/lib/common/Urls.dart
import 'package:tiq/services/GStateProvider.dart';

class Urls {
  Urls._internal();
  static final Urls _singleton = new Urls._internal();
  static Urls get instance => _singleton;

  final String _timeRead =                        'time/read';
  final String _userLogin =                       'auth/login';
  final String _facilityListSimpleReadByPerson =  'facility/list/simple/by_person';
  final String _facilityConfigGet =               'facility/config/get';
  final String _pacHeadReadByBarcode =            'pac/head/read';
  final String _purchaseTaskListByPerson =        'purchase/task/list/by_person';
  final String _purchaseTaskCreateFromPacs =      'purchase/task/create/from_pacs';
  final String _purchaseTaskUpdateRead =          'purchase/task/update/read';
  final String _purchaseTaskLineReadByBarcode =   'purchase/task/line/read/by_barcode';
  final String _purchaseTaskLineUpdatePost =      'purchase/task/line/update/post';
  final String _purchaseTaskFinish =              'purchase/task/finish';

  String get timeReadUrl => GStateProvider.instance.settingsApp.wmsServiceBaseAddress + _timeRead;
  String get userLoginUrl => GStateProvider.instance.settingsApp.wmsServiceBaseAddress + _userLogin;
  String get facilityListSimpleReadByPersonUrl => GStateProvider.instance.settingsApp.wmsServiceBaseAddress + _facilityListSimpleReadByPerson;
  String get facilityConfigGetUrl => GStateProvider.instance.settingsApp.wmsServiceBaseAddress + _facilityConfigGet;
  String get pacHeadReadByBarcodeUrl => GStateProvider.instance.settingsApp.wmsServiceBaseAddress + _pacHeadReadByBarcode;
  String get purchaseTaskListForPersonUrl => GStateProvider.instance.settingsApp.wmsServiceBaseAddress + _purchaseTaskListByPerson;
  String get purchaseTaskCreateFromPacsUrl => GStateProvider.instance.settingsApp.wmsServiceBaseAddress + _purchaseTaskCreateFromPacs;
  String get purchaseTaskUpdateReadUrl => GStateProvider.instance.settingsApp.wmsServiceBaseAddress + _purchaseTaskUpdateRead;
  String get purchaseTaskLineReadByBarcodeUrl => GStateProvider.instance.settingsApp.wmsServiceBaseAddress + _purchaseTaskLineReadByBarcode;
  String get purchaseTaskLineUpdatePostUrl => GStateProvider.instance.settingsApp.wmsServiceBaseAddress + _purchaseTaskLineUpdatePost;
  String get purchaseTaskFinishUrl => GStateProvider.instance.settingsApp.wmsServiceBaseAddress + _purchaseTaskFinish;
}